src/styles/blocks/search-bar.css

```diff
@@ -0,0 +1,15 @@
+.search-bar {
+  display: flex;
+  align-items: center;
+  border: 1px solid rgb(225, 231, 237);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  margin-top: 20px;
```

Comment:
Be consistent with your margins. It's better to add only top or bottom margins to avoid potential margin collapse issues. If the design allows it, consider using only `margin-bottom` or only `margin-top` for spacing between elements.